drivers/hmis/app/models/hmis/hud/processors/service_processor.rb

```diff
@@ -46,12 +46,6 @@ def assign_metadata
       hmis_service = @processor.owner_factory
       hmis_service.assign_attributes(user: @processor.hud_user)
       hmis_service.owner.assign_attributes(user: @processor.hud_user)
-      return unless hmis_service.hud_service?
-
-      hmis_service.owner.assign_attributes(
-        record_type: hmis_service.service_type&.hud_record_type,
-        type_provided: hmis_service.service_type&.hud_type_provided,
-      )
```

Comment:
Moved this logic to the HmisService initializer. We don't let you change the service type after the fact, so it doesn't need to be here anymore